src/commands/ExecCommand.js

```diff
@@ -27,6 +27,14 @@ export const builder = {
     type: "boolean",
     default: undefined,
   },
+  // This option disables prefix for stream output so that the log can be handled by
+  // tools like Visual Studio Code
+  "disable-stream-prefix": {
```

Comment:
Can we rename this `--prefix`, so the auto-negated `--no-prefix` works, as well as the lerna.json-based `"prefix": false` works? `--prefix` certainly has different meanings in other contexts, but it's pretty safe for `lerna exec` and `lerna run`.

You'll also need to add `prefix: true,` to the `get defaultOptions()` block in ExecCommand and RunCommand.